extrabol/extrabol.py

```diff
@@ -890,23 +890,27 @@ def write_output(lc, dense_times, dense_lc, Tarr, Terr_arr, Rarr, Rerr_arr,
     '''
 
     dense_lc = np.reshape(dense_lc, (len(dense_lc), -1))
-    dense_lc = np.hstack((np.reshape(-dense_times, (len(dense_times), 1)), dense_lc))
-    tabledata = np.stack((Tarr / 1e3, Terr_arr / 1e3, Rarr / 1e15,
-                          Rerr_arr / 1e15, np.log10(bol_lum),
-                          np.log10(bol_err))).T
-    tabledata = np.hstack((-dense_lc, tabledata)).T
-    table_header = ['Phase']
+    dense_lc = np.hstack((np.reshape(dense_times, (len(dense_times), 1)), dense_lc))
+    tabledata = np.stack((Tarr, Terr_arr, Rarr, Rerr_arr, bol_lum, bol_err)).T
+    tabledata = np.hstack((dense_lc, tabledata)).T
+
+    ufilts = np.unique(my_filters)
+    table_header = []
+    table_header.append('Time (MJD)')
```

Comment:
This is merged the wrong way. Lines 896-899 should be replaced by:
```
table_header = ['Phase']
```

Reply:
thanks! I pushed a fix